Logitar/Logitar-UI
# Project base setup

# Project base setup

Covers and resolves #1 

## How to review this merge request

Using at least Node.js `14.19.0`:

1. Clone the repository and run `npm i` inside.
2. Run `npm run test` and make sure all tests pass.
3. Run `npm run storybook` and wait for a navigator window to open. Once loaded, it should have a single story with an unstyled button inside.
4. Change directory to the example project: `cd ./example`
5. Run `npm i`
6. Change directory to the root: `cd ../`
7. Run `npm run dev`, click on the link in the console or navigate to `http://localhost:5173`
8. You should see a default Vite.js React app!

## Moving forward

Once all is done, we should be able to merge to `main`, release package `@logitar/logitar-ui` version `0.0.0` on `npm` and I will be able to setup the repository so that a merge request will require all tests to pass.
